app/services/bulk_tagging/build_tag_migration.rb

```diff
@@ -2,20 +2,16 @@ module BulkTagging
   class BuildTagMigration
     class InvalidArgumentError < ArgumentError; end
 
-    attr_reader :tag_migration_params, :taxon_content_ids, :content_base_paths
+    attr_reader :source_content_item, :taxon_content_ids, :content_base_paths
 
-    def initialize(tag_migration_params:, taxon_content_ids:, content_base_paths:)
-      @tag_migration_params = tag_migration_params
+    def initialize(source_content_item, taxon_content_ids, content_base_paths)
```

Comment:
I think you should keep the signature of the method, its easier to read when you specify which parameter is which